openmc/mgxs/mgxs.py

```diff
@@ -368,6 +373,11 @@ def nuclides(self, nuclides):
         cv.check_iterable_type('nuclides', nuclides, basestring)
         self._nuclides = nuclides
 
+    @estimator.setter
+    def estimator(self, estimator):
```

Comment:
If my comment about restricting the estimator to only analog and collision for some MGXS is well founded, then this property should check for only that subset of types for those subclasses.  Perhaps this property needs to be redefined a few places, or there could be a new `_accepted_types` class attribute for type checking.